resources/js/app.js

```diff
@@ -122,20 +86,26 @@ function init() {
                 }
             }
         },
+        computed: {
+            // Wrap the local storage in getter and setter functions so you do not have to interact using .value
+            guestEmail: wrapValue(useLocalStorage('email', (window.debug ? 'wayne@enterprises.com' : ''), {serializer: StorageSerializers.string})),
+        },
         asyncComputed: {
             swatches () {
-                if (localStorage.swatches) {
-                    return JSON.parse(localStorage.swatches);
+                if (Object.keys(swatches.value).length !== 0) {
+                    return swatches;
                 }
 
                 return axios.get('/api/swatches')
                     .then((response) => {
-                        localStorage.swatches = JSON.stringify(response.data)
-                        return response.data
+                        swatches.value = response.data
+
+                        return swatches
                     })
                     .catch((error) => {
+                        console.error(error)
```

Comment:
This isn't needed anymore since #186 right?

Reply:
#186 logs the customer-facing messages in the console, while this console.error will log the true error that occurred